waiter/src/waiter/core.clj

```diff
@@ -632,6 +633,8 @@
                        router-id]
                 (discovery/register router-id curator name (str base-path "/" discovery-relative-path)
                                     {:host host :port (primary-port port)}))
+   :event-chan (pc/fnk [[:settings [:event-maintainer-config event-chan-buffer-size]]]
```

Comment:
This should not be necessary to expose. The state can be exposed in a `query-state-fn` and we will use a new state endpoint to display the state.